### DISCUSSION TITLE
How can I change from network of 10.0.0.2 to 192.168.88.2

### DISCUSSION THREAD

**Discussion by realChytraeus** [quickemu-project/quickemu#840] [Quickemu]
#### Expected behaviour
I have a netbsd-9.2 virtual machine I added with quickgui.  In the netbsd-9.2/ folder I added the following to netbsd-9.2.sh after -netdev.
192.168.88.1/24. This changed nothing so I removed 192.168.88.1/24. 
I tried and tested adding the option of -netdev to netbsd-9.2.conf . I added the following line to it:
-netdev="192.168.88.1/24"

#### Actual behaviour
I restarted the VM and using ifconfig I see that I still have 10.0.2.15.
I would like to be able to have all Linux and BSD VM's and other OS's on my local network of 192.168.88.2.

#### Steps to reproduce the behaviour

see above.

#### Quickemu output
quickemu  --vm netbsd-9.2.conf 
netbsd-9.2.conf: line 2: -netdev=192.168.88.1/24: No such file or directory
Quickemu 4.5 using /usr/bin/qemu-system-x86_64 v7.0.0
 - Host:     Ubuntu 22.10 running Linux 5.19 (bernard)
 - CPU:      Intel(R) Core(TM) i7-4710MQ CPU @ 2.50GHz
 - CPU VM:   1 Socket(s), 2 Core(s), 2 Thread(s), 6G RAM
 - BOOT:     Legacy BIOS (Netbsd)
 - Disk:     netbsd-9.2/disk.qcow2 (80G)
 - Display:  SDL, qxl-vga, GL (on), VirGL (off)
 - ssh:      On host:  ssh user@localhost -p 22220
 - WebDAV:   On guest: dav://localhost:9843/
 - 9P:       On guest:  - Network:  User (virtio-net)
 - Monitor:  On host:  nc -U "netbsd-9.2/netbsd-9.2-monitor.socket"
             or     :  socat -,echo=0,icanon=0 unix-connect:netbsd-9.2/netbsd-9.2-monitor.socket
 - Serial:   On host:  nc -U "netbsd-9.2/netbsd-9.2-serial.socket"
             or     :  socat -,echo=0,icanon=0 unix-connect:netbsd-9.2/netbsd-9.2-serial.socket
 - Process:  Starting netbsd-9.2.conf as netbsd-9.2 (90327)

#### Linux Distribution & Kernel

lsb_release --all
LSB Version:    core-11.2ubuntu1-noarch:printing-11.2ubuntu1-noarch:security-11.2ubuntu1-noarch
Distributor ID: Ubuntu
Description:    Ubuntu 22.10
Release:        22.10
Codename:       kinetic
~ $ uname -a
Linux bernard 5.19.0-1015-lowlatency #16-Ubuntu SMP PREEMPT_DYNAMIC Thu Jan 5 20:11:09 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux

**Comment by philclifford** [comment]
To pass extra arguments to `qemu` you need to add `--extra-args` to your config file or command line.
You'll also probably need to quote arguments especially where things like "/" appear (see your error message above).  You need to look at the `qemu` manual to craft the `--netdev` you need to pass, but it looks to me that you at least need something like:
```
--extra-args='-netdev user,id=mynetid,net=192.168.88.1/24, ...
```
based on the `-netdev` entry in the <VM>/<VM>.sh that `quickemu` generates.  You can tweak and re-run that generated bash script until you get the incantation right but remember `quickemu` will regenerate it so you need to adapt your config file.
